packages/ui-extensions/src/surfaces/admin/components/Card/Card.ts

```diff
@@ -0,0 +1,13 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import {BackgroundProps} from '../shared';
+
+export interface CardProps extends BackgroundProps {
+  /**
+   * Adjust the padding of all edges.
+   *
+   * `true` applies a default padding that is appropriate for the component.
+   */
+  padding?: boolean;
```

Comment:
Is it true by default?

Reply:
It shouldn't. i reworded the comment above in case it was unclear